src/manage-apps/manage-repos-repoman.md

```diff
@@ -0,0 +1,92 @@
+# Managing Repositories with Repoman
+
+Repoman allows users to add apt and Flatpak repositories. Users may want to add an apt repository or additional Flatpak repository if their software is not included in the Pop!\_Shop, or they may want to remove an unneeded repository after uninstalling the only application that required it.
+
+Access Repoman by launching the Pop!\_Shop, clicking the three lines, and then clicking `System Software Sources...`.
+
+![Access Repoman](/images/manage-repos/access-repoman.png)
+
+>**Caution**: Adding third-party apt or Flatpak repositories allows installation of software that has not been validated by System76 or other trusted Linux communities. Software available via third-party apt and Flatpak repositories may not be vetted against packages that introduce security vulnerabilities. Users should take caution and only add external repositories that they trust.
+---
+
+## Official Sources
+
+This section lists System76's Ubuntu mirror. This mirror includes Pop!\_OS application packages that come from Ubuntu (additional packages that make Pop!\_OS are added on top via the `apt.pop-os.org/release` repo found in the `Extra Sources` tab). This is also where installation files for many applications in the Pop!\_Shop reside. Changing this mirror is not necessary since Pop!\_OS will automatically download updates from the fastest source using System76's worldwide CDN.
+
+The toggle switches enable or disable software components based on licensing and support:
+
+- **Restricted** - Proprietary drivers and components that are not open source
+- **Universe** - Free and open-source software that is community maintained
+- **Multiverse** - Software restricted by copyright or legal reasons
+
+![Official Sources Tab](/images/manage-repos/official-sources-tab.png)
+
+## Managing apt Repositories with Repoman
+
+Apt archives (including [PPAs](manage-repos.md#ppas)) are created by developers to distribute software not included in default Ubuntu or Pop!\_OS repositories.
+
+### Viewing Extra Sources
+
+Click on the `Extra Sources` tab to display repositories used by third-party applications.
+
+![Extra Sources](/images/manage-repos/extra-sources.png)
+
+>**Note:** While this page is generally used to add sources for third-party apt repositories, most sources with a URI containing `http://apt.pop-os.org` are critical to Pop!\_OS's desktop functionality and should not be removed. [See this page](https://apt-origin.pop-os.org/) for more information about Pop!\_OS's apt sources.
```

Comment:
@Ampersandstorm Was there a reason you linked to `apt-origin.pop-os.org` instead of `apt.pop-os.org`? `apt-origin` goes directly to the web server without going through the CDN. The webpage isn't going to use very much bandwidth, but I don't see why `apt-origin` needs to be exposed here when `apt` also serves the webpage.

Reply:
No reason, I'll change it to `apt.pop-os.org`.